idaes/generic_models/properties/core/eos/ceos.py

```diff
@@ -82,12 +85,25 @@ class MixingRuleB(Enum):
         CubicType.SRK: {'u': 1, 'w': 0, 'omegaA': 0.42748, 'coeff_b': 0.08664}
         }
 
+eps_SL = 1E-8
 
 CubicConfig = ConfigBlock()
 CubicConfig.declare("type", ConfigValue(
     domain=In(CubicType),
     description="Equation of state to use",
     doc="Enum indicating type of cubic equation of state to use."))
+CubicConfig.declare("mixing_rule_a", ConfigValue(
```

Comment:
On the user's end, it should be exactly the same. From the perspective of the class, the mixing rules were never made available globally, but only checked for when defining am and bm:
```
        def rule_am(m, p):
            try:
                rule = m.params.get_phase(p).config.equation_of_state_options[
                    "mixing_rule_a"]
            except (KeyError, TypeError):
                rule = MixingRuleA.default

            a = getattr(m, cname+"_a")
            if rule == MixingRuleA.default:
                return rule_am_default(m, cname, a, p)
            else:
                raise ConfigurationError(
                    "{} Unrecognized option for Equation of State "
                    "mixing_rule_a: {}. Must be an instance of MixingRuleA "
                    "Enum.".format(m.name, rule))
        b.add_component(cname+'_am',
                        Expression(b.phase_list, rule=rule_am))
```
However, I wanted to make the dependency of the fugacity coefficient on the mixing rules explicit and the bubble and dew property methods didn't even check for mixing rules, so attaching it to the parameter block made sense. I'm not completely consistent, because I also attach it to the phase object (perhaps in anticipation of permitting different mixing rules in different phases).